packages/browsers/src/install.ts

```diff
@@ -459,7 +459,7 @@ export async function canDownload(options: InstallOptions): Promise<boolean> {
   );
 }
 
-function getDownloadUrl(
+export function getDownloadUrl(
```

Comment:
Let's add a jsdoc marking it with `@public` and a short description about what the function does